### PR TITLE
added last_link_flapped to nxos_show_interface

### PR DESCRIPTION
**Pull request by aSauerwein**
##### ISSUE TYPE

 - New Template Pull Request

##### COMPONENT

cisco_nxos_show_interface.template

##### SUMMARY
added the value "last_link_flapped" to the tamplate

### CODE REVIEW

**Comment by jmcgill298** [comment]
@aSauerwein please add the original test files back and add the new `las_link_flapped` key/value pair to the `parsed` files

**Comment by aSauerwein** [comment]
> Need to restore existing test files

sorry. just restored the old test files. 
hope it's fine now

File: templates/cisco_nxos_show_interface.template [hunk 1]
```diff
@@ -16,6 +16,7 @@ Value OUTPUT_ERRORS (\d+)
 Value BANDWIDTH (\d+\s+\w+)
 Value DELAY (\d+\s+\w+)
 Value ENCAPSULATION (\w+)
+Value LAST_LINK_FLAPPED (.+)
```
Comment (jmcgill298) [inline templates/cisco_nxos_show_interface.template]:
```suggestion
Value LAST_LINK_FLAPPED (.+?)
```

File: templates/cisco_nxos_show_interface.template [hunk 2]
```diff
@@ -32,3 +33,4 @@ Start
   ^\s+${INPUT_ERRORS}\s+input\s+error\s+\d+\s+short\s+frame\s+\d+\s+overrun\s+\d+\s+underrun\s+\d+\s+ignored\s*$$
   ^\s+${OUTPUT_PACKETS}\s+output\s+packets\s+\d+\s+bytes\s*$$
   ^\s+${OUTPUT_ERRORS}\s+output\s+error\s+\d+\s+collision\s+\d+\s+deferred\s+\d+\s+late\s+collision\s*$$
+  ^\s+Last\s+link\s+flapped\s+${LAST_LINK_FLAPPED}
```
Comment (jmcgill298) [inline templates/cisco_nxos_show_interface.template]:
```suggestion
  ^\s+Last\s+link\s+flapped\s+${LAST_LINK_FLAPPED}\s*$$
```

**Pull request merged**